### ISSUE TITLE
Team tagline

### ISSUE THREAD

**Issue by gadenbuie** [GerkeLab/gerklab-com#1]
We need a tagline that isn't 

> Doing research and fighting the good fight against cancer.

but is roughly this long.

**Comment by tgerke** [comment]
Something like the following?
> Accelerating cancer epidemiology through modern data science
Arguably, cancer epidemiologists have been "data scientists" forever, but I mean to distinguish what we do (e.g. contemporary R/Shiny practice) versus what is common and more traditional in the field (SAS). Maybe appending practice at the end?
> Accelerating cancer epidemiology through modern data science practice
Maybe this is getting too wordy though...

**Comment by gadenbuie** [comment]
I like it without "practice" and "with" instead of "through"...

<details><summary>Accelerating cancer epidemiology with modern data science.</summary>
<img width="609" alt="image" src="https://user-images.githubusercontent.com/5420529/42325863-7afe4312-8035-11e8-907a-fcc08760ffb1.png"></details><br>

What about adding the word "research"?

<details><summary>Accelerating cancer epidemiology research with modern data science.</summary>
<img width="670" alt="image" src="https://user-images.githubusercontent.com/5420529/42325952-ae635288-8035-11e8-839d-c157fd31927f.png">
</details><br>

I'm not the biggest fan of adding "research" but it feels more balanced visually and in terms of the number of words/syllables.

**Comment by tgerke** [comment]
Great with adding research. It makes a lot more sense, but I felt like I was bloating the word count when I was inserting it in my versions. 
* Accelerating cancer epidemiology research with modern data science.
it is?
